packages/create-docusaurus/templates/classic/docusaurus.config.js

```diff
@@ -14,7 +14,7 @@ const config = {
   favicon: 'img/favicon.ico',
 
   // Set the production url of your site here
-  url: 'https://your-docusaurus-test-site.com',
+  url: 'https://your-docusaurus-site.example',
```

Comment:
Maybe use `.example.com`? Seems more unsurprising.

Reply:
@Josh-Cena Not so terrible, but it looks longer than `.example`.
A candidate is `your-docusaurus-site.example.com` or `docusaurus-site.example.com`.

https://example.com & https://www.example.com are resolved to an example page by IANA, but pages for the other subdomains  don't seem to be prepared.

Reply:
It's okay if the page is a broken page. And yes, I meant `your-docusaurus-site.example.com`.

Reply:
Ok, I'll rename all of them.

Reply:
@Josh-Cena Done.